source_handlers_test.go

```diff
@@ -205,16 +330,28 @@ func TestSourceTypeSourceSubcollectionList(t *testing.T) {
 		if !ok {
 			t.Error("model did not deserialize as a source")
 		}
+	}
+
+	sourcesBelongToTenant, err := checkAllSourcesBelongToTenant(tenantId, out.Data)
+	if err != nil {
+		t.Error(err)
+	}
 
+	if !sourcesBelongToTenant {
+		t.Error("the tenant doesn't own all sources")
 	}
```

Comment:
Maybe you don't need a boolean value being returned from the `checkAllSourcesBelongToTenant` function. You could simply make it return `err` if they don't belong, and you just have to keep the `if err != nil` check. What do you think?

Reply:
good point, thanks ... solved in new commit